sparsEDA.py

```diff
@@ -395,6 +395,9 @@ def sparsEDA(signalIn,sr,epsilon,Kmax,dmin,rho):
 
     # PP
     ind = np.argwhere(SCRaux > 0).reshape(-1)
+    driver = np.zeros(len(SCRaux))
+    if not ind.shape[0]:
```

Comment:
Can you please change the if statement to the following? Thanks!

if ind.size == 0

Reply:
Can do